src/behaviour.rs

```diff
@@ -104,7 +105,7 @@ impl Behaviour {
             ConfigBuilder::default()
                 .max_transmit_size(MAX_TRANSMIT_SIZE * MEGABYTES) // block size is limitted to 2 MB. Multiply by factor of safety = 2.
                 .message_id_fn(build_msg_id)
-                .heartbeat_interval(Duration::from_secs(heartbeat_secs))
+                .heartbeat_interval(Duration::from_secs(10))
```

Comment:
I think we can take this magic number out and put them with the constant up there. (Not a must, just think it might be cleaner)